test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -6278,10 +6281,11 @@ BOOST_AUTO_TEST_CASE(invalid_address_length_long)
 			}
 		}
 	)";
-	CHECK_ALLOW_MULTI(text, (std::vector<std::pair<Error::Type, std::string>>{
-		{Error::Type::Warning, "This looks like an address but has an invalid checksum."},
-		{Error::Type::TypeError, "not implicitly convertible"}
-	}));
+	vector<pair<Error::Type, std::string>> expectedErrors = {
+			{Error::Type::SyntaxError, "This looks like an address but is not exactly 20 bytes long."},
+			{Error::Type::TypeError, "not implicitly convertible"}
+		};
+	CHECK_ALLOW_MULTI(text, expectedErrors);
 }
```

Comment:
Do we have a test where the type conversion from integer to address is actually tested?

Reply:
The test below us for address to integer, what about integer to address?